### ISSUE TITLE
Outreachy blog post: Project progress report

### ISSUE THREAD

**Issue by jwflory** [unicef/inventory-hugo-theme#140]
> For this week's blog post, we'd like to write a progress report of what you've accomplished in the first half of the internship. The blog post should also detail what your modified goals for the second half of the internship is.
>
> Here's some prompts that could get you started writing this blog post:
>
> - What was your original internship project timeline?
> - What goals have you met?
> - What have you accomplished in the first half of your internship?
> - What project goals took longer than expected?
> - Why did those project goals take longer than expected?
> - What would you do differently if you were starting the project over?
> - Which original goals needed to be modified?
> - What is your new plan for the second half of the internship?

**Comment by jwflory** [comment]
The blog post draft is finished, but we were talking baout publishing. Instead of the Root blog, @Neha9849 will try out some of the hosted platforms (e.g. Substack, Medium, etc.) and post the blog there. Once it is up, we'll share it on social media and with the team! 😀

**Comment by jwflory** [comment]
The blog post is published here:

https://root-blogging-app.herokuapp.com/blog/62dd17f2ecb39aa2e8d84cfd

I am closing this as complete, although there are a few more steps before we can share and publish this widely with the support of UNICEF communications. The default landing page needs to be customized from the Root default and the site publish under a dedicated domain instead of the default Heroku URL.

https://root-blogging-app.herokuapp.com/

**Comment by jwflory** [comment]
Also saved in the Internet Archive for posterity:

https://web.archive.org/web/20220728131231/https://root-blogging-app.herokuapp.com/blog/62dd17f2ecb39aa2e8d84cfd
